shaders/src/light_punctual.fs

```diff
@@ -106,8 +106,11 @@ float getSquareFalloffAttenuation(float distanceSquare, float falloff) {
 float getDistanceAttenuation(const highp vec3 posToLight, float falloff) {
     float distanceSquare = dot(posToLight, posToLight);
     float attenuation = getSquareFalloffAttenuation(distanceSquare, falloff);
+    // light far attenuation
+    float d = length(getWorldPosition() - getWorldCameraPosition());
```

Comment:
We already have the result of that subtraction, it's `posToLight`. Also we have the square distance above, just sqrt() it instead of calling length (or better, work in square distance if you can make it work?)

Reply:
No, `posToLight` is not the same thing -- it's the position of the fragment to the light (`positionFalloff.xyz - worldPosition`). I need the distance from the camera to the fragment.